dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -762,15 +767,16 @@ def __init__(self, pipeline, **kwargs):
                 if disallowed_kwarg in kwargs.keys():
                     raise TypeError((
                         "__init__() got an unexpected keyword argument '{}'. "
-                        "Dataset inputs are allowed only in 'experimental.DALIDatasetWithInputs'.").format(
-                        disallowed_kwarg))
+                        "Dataset inputs are allowed only in"
+                        " 'experimental.DALIDatasetWithInputs'.").format(disallowed_kwarg))
             # We detected External Source nodes in the Pipeline
             if _has_external_source(pipeline):
                 raise ValueError(("DALIDataset got a DALI pipeline containing External Source "
-                    "operator nodes. External Source nodes can be used to express placeholders "
-                    "for tf.data.Dataset inputs to DALI or to run user-provided Python code "
-                    "via `source` parameter. Support for Dataset inputs and External Source's "
-                    "`source` is allowed only in 'experimental.DALIDatasetWithInputs'."))
+                                  "operator nodes. External Source nodes can be used to express "
+                                  "placeholders for tf.data.Dataset inputs to DALI or to run "
+                                  "user-provided Python code via `source` parameter. Support for "
+                                  "Dataset inputs and External Source's `source` is allowed only "
+                                  "in 'experimental.DALIDatasetWithInputs'."))
```

Comment:
Remove superfluous parenthesis:
```suggestion
                raise ValueError("DALIDataset got a DALI pipeline containing External Source "
                                 "operator nodes. External Source nodes can be used to express "
                                 "placeholders for tf.data.Dataset inputs to DALI or to run "
                                 "user-provided Python code via `source` parameter. Support for "
                                 "Dataset inputs and External Source's `source` is allowed only "
                                 "in 'experimental.DALIDatasetWithInputs'.")
```